spec/requests/stories_index_spec.rb

```diff
@@ -1,5 +1,15 @@
 require "rails_helper"
 
+RSpec.shared_examples "redirects to the lowercase route" do
+  context "when a path contains uppercase characters" do
+    it "redirects to the lowercase route" do
+      get path
+      expect(response).to have_http_status(:moved_permanently)
+      expect(response).to redirect_to(path.downcase)
+    end
+  end
+end
```

Comment:
Nice use of shared examples. Another option is also to pass path in as a parameter `...do |path|` so that you don't need to use a block (and you can use `it_behaves_like`).

But that's more just a ProTip; I think what you have works just fine 👍

Reply:
Thanks! I actually had it that way initially but that setup didn't read as well or make as much sense for this particular spec, so I opted to set it up this way. 😄

Reply:
Yep, I think this does work better because of the `build` stuff.